### PR TITLE
ConfigurationGathering.py:default_sections to base_sections

### PR DESCRIPTION
**Pull request by ankit01ojha**
Fixes https://github.com/coala/coala/issues/3395

### Checklist

- [ ] I have rebased properly. Please see [our tutorial on rebasing](http://coala.io/git#rebasing).
- [ ] I have gone through the [commit guidelines](http://coala.io/commit) and I've followed them.
- [ ] I have followed the [guidelines for the commit shortlog](http://coala.io/commit#shortlog).
- [ ] I have followed the [guidelines for the commit body](http://coala.io/commit#commit-body).
- [ ] I have included the issue URL with the 'Fixes' keyword if the commit fixes a *real bug* and the 'Closes' keyword if it adds a feature or enhancement. For details, check the [issue reference section in our commit guidelines](http://coala.io/commit#issue-reference).
- [ ] I have [run all the tests](http://api.coala.io/en/latest/Developers/Executing_Tests.html) and they all pass. If any CI (below) is not green, please fix them. If GitMate issues appear you will have to amend your commits, pushing new commits on top is not sufficient!

### CODE REVIEW

**Comment by sims1253** [comment]
Hey,
You need to shorten your shortlog and also there is a whitespace missing. The filename is long but giving the full filename as an identifier is not strictly needed. Maybe use something shorter that points into the right direction.
The issue you want to close is not a bug so you should use the Closes keyword not Fixes.
See coala.io/commits for more info.

**Comment by ankit01ojha** [comment]
@sims1253 Can suggest what should I write in place of filename so that it would point in write direction.

**Comment by hemangsk** [comment]
In that case, you can write the name of package, `settings: Change default_sections to base_sections`, just don't  forget to add the file name in shortlog.

File: coalib/settings/ConfigurationGathering.py [hunk 1]
```diff
@@ -163,15 +163,14 @@ def load_configuration(arg_list, log_printer, arg_parser=None):
...
-                                            log_printer)
+        base_sections = load_config_file(Constants.system_coafile,  log_printer)
         user_sections = load_config_file(
             Constants.user_coafile,
             log_printer,
             silent=True)
 
         default_config = str(
-            default_sections['default'].get('config', '.coafile'))
+            base_sections['default'].get('config', '.coafile'))
```
Comment (vijeth-aradhya) [inline coalib/settings/ConfigurationGathering.py]: Can this be written in one line ( <= 80 chars )? If so, please do :)

**Comment by fneu** [comment]
@ankit01ojha There is also #3533  still open by you and it appears that both PRs address the same issue.

Are you aware that you can update a PR by (force) pushing to the respective branch?

In any way please close one of them (probably this one)

**Pull request closed without merging**